Model/Api/DiscountCodeValidation.php

```diff
@@ -202,7 +202,7 @@ public function validate() {
                 'X-Bolt-Plugin-Version' => $this->configHelper->getModuleVersion()
             ]);
 
-            $this->hookHelper->verifyWebhook();
+            //$this->hookHelper->verifyWebhook();
```

Comment:
remove if not needed.

Reply:
it's needed but left commented out while I was doing test from a REST client.